package.json

```diff
@@ -125,6 +160,22 @@
 				"title": "Scan for refactorings",
 				"category": "Sourcery"
 			},
+			{
+				"command": "sourcery.scan.rule",
+				"title": "Scan with specific rule",
+				"category": "Sourcery"
+			},
+			{
+				"command": "sourcery.apply.rule",
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Call this `sourcery.scan.applyRule` so it's clear it's part of the scan?